File: web3rpc/rpc-specs/code-samples/python/eth/block/getBlockTransactionCountByNumber.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

blockNumber = "0xe8"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_block_transaction_count_by_number(blockNumber)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/block/getBlockTransactionCountByNumber.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const blockNumber = "0xe8";
  provider.eth
    .getBlockTransactionCountByNumber(
      blockNumber,
      {},
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
